### ISSUE TITLE
Add option for which direct to Render in

### ISSUE THREAD

**Issue by navarr** [crafatar/crafatar#90]
Specifically for the /renders/ endpoints.  Default would remain Southeast.  Would be nice to at least render Southwest as well - but some may also desire the ability to render Northeast and Northwest.

**Comment by jomo** [comment]
:+1: I've had this in my mind already. This is currently low priority, so it might take a while until we implement this.

PRs always welcome :)

**Comment by MelonCola** [comment]
Seeing as this is 5 years old, I'm just curious, is this still being considered? Would be pretty helpful to have.

**Comment by jomo** [comment]
Yes, but it might take another five years if nobody sends a PR ;)

If you want the rendered players to look left instead of looking right, you can flip the image with CSS. Of course it's not exactly the same for asymmetrical skins, but good enough until changing the direction is an option.
